src/Data/Vec/Base.agda

```diff
@@ -9,10 +9,10 @@
 module Data.Vec.Base where
 
 open import Data.Bool.Base using (Bool; true; false; if_then_else_)
-open import Data.Nat.Base
+open import Data.Nat.Base as N
```

Comment:
Do you need the qualified import? If you do, convention/style dictates it should be imported as either `Nat` or `ℕ`.